pkg/utils/time.go

```diff
@@ -108,3 +71,58 @@ func getBootTimeInJiffies() int64 {
 
 	return 0
 }
+
+//
+// Boot time functions
+//
+
+// GetStartTimeNS returns the system start time in nanoseconds (using CLOCK_MONOTONIC).
```

Comment:
But what is start time?
When I read it, start time is the time Tracee start.
Boot time is the time of the boot.
This way they are both on the same logic of names.
However, start time is the time since boot, and boot time is the time of boot since epoch.
These 2 numbers are very different in their meaning, so the names have to clarify it.